packages/devtools/package.json

```diff
@@ -11,6 +11,7 @@
 	},
 	"license": "MIT",
 	"dependencies": {
+		"@josepot/rxjs-utils": "^0.15.2",
```

Comment:
:joy: At some point we should upgrade the dependencies to use `@react-rxjs/core` and `@react-rxjs/utils`, though :smile: 

I don't think that using the personal utils of a random user is a good idea :sweat_smile:

Reply:
Is there any license fee? :'D